src/axom/sidre/core/Group.cpp

```diff
@@ -2348,6 +2350,23 @@ bool Group::importConduitTreeExternal(conduit::Node &node,
   return success;
 }
 
+/*
+ *************************************************************************
+ *
+ * Exports this Group into a Conduit node tree.
+ * This exports a generic conduit tree, not one with sidre conventions.
+ *
+ *************************************************************************
+ */
+
+conduit::Node Group::exportConduitTree() 
+{
+  conduit::Node result;
```

Comment:
FYI: Returning with this style will invoke lots of copies (including copies of external data) - so the style we typically use is to pass in a ref to a Node.